CHANGELOG.md

```diff
@@ -1,5 +1,43 @@
 # Change Log
 
+## [1.0.0rc1](https://github.com/stellargraph/stellargraph/tree/v1.0.0rc1)
+
+[Full Changelog](https://github.com/stellargraph/stellargraph/compare/v0.11.0...v1.0.0rc1)
+
```

Comment:
Since this is a 1.0 release candidate, I think it could be worth an extra sentence, eg:
"This is the first release candidate for StellarGraph 1.0, and a major milestone of a full release version of the library following 2 years of active development"